<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeDomainSettingDialog</class>
 <widget class="QDialog" name="TimeDomainSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>186</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="TransmissionBitLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>传输位宽</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="TracksNumLabel_2">
       <property name="text">
        <string>采样帧长</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="FrameLenghtBox">
       <property name="currentText">
        <string>1.5ms</string>
       </property>
       <item>
        <property name="text">
         <string>1.5ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4.0ms</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="SampleBitLabel">
       <property name="text">
        <string>样点类型</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="TransmissionBitBox">
       <property name="currentText">
        <string>8</string>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="TracksNumLabel">
       <property name="text">
        <string>音轨数量</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="SampleBitBox">
       <property name="currentText">
        <string>int8</string>
       </property>
       <item>
        <property name="text">
         <string>int8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>float</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="SamplingRateLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>采样率  </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="TracksNumBox">
       <property name="currentText">
        <string>1</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="SamplingRateBox">
       <property name="currentText">
        <string>8000Hz</string>
       </property>
       <item>
        <property name="text">
         <string>8000Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11025Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12000Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16000Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22050Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24000Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32000Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>44100Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48000Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64000Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>88200Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>96000Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>176400Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>192000Hz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="TimeConfirmBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
